### PR TITLE
KIALI-2529 Remove absolute imports

### PR DESCRIPTION
**Pull request by cfcosta**
Moving to `react-scripts` from `react-scripts-ts` moves our
transpilation layer to use babel instead of tsc, which forbids absolute
imports.

### CODE REVIEW

**Comment by josejulio** [comment]
Ah.. I liked the absolute imports.. :(

**Comment by cfcosta** [comment]
> Ah.. I liked the absolute imports.. :(
> It felt it was easier to locate the resource we were importing.

You can use "go to definition" on the import, all editors should implement this :)
Also, the next step for us might be adding "paths" to transform something like:

```typescript
import { Foo } from '../../../types/foo';
```

Into:

```typescript
import { Foo } from 'types/foo';
```

File: tsconfig.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "baseUrl": "./",
+    "baseUrl": "src",
```
Comment (israel-hdez) [inline tsconfig.json]: is this change also needed?
Looks redundant if relative paths are being used now... Perhaps, is it to forbid absolute imports?

Comment (cfcosta) [reply tsconfig.json]: That’s a nice side effect, but it does not compile after the upgrade.

Comment (israel-hdez) [reply tsconfig.json]: Ok, I'm approving.

**Pull request merged**